resource/build-scripts/build-config-file.js

```diff
@@ -1,15 +1,29 @@
-'use strict';
-/**
- * External Dependencies
- */
-var base = require( '../../desktop-config/config-base.json' );
-var config = require( process.argv[2] );
+// TODO: evaluate if there is a more straightforward way to create config files e.g. how does calypso solve this?
```

Comment:
Yes please! https://github.com/Automattic/wp-calypso/tree/master/server/config and https://github.com/Automattic/wp-calypso/blob/master/client/lib/create-config/index.js seem relevant 🙂